frontend/micro-ui/web/micro-ui-internals/packages/modules/hearings/src/pages/employee/InsideHearingMainPage.js

```diff
@@ -223,7 +223,7 @@ const InsideHearingMainPage = () => {
       deposition: witnessDepositionText,
     });
     _updateTranscriptRequest({ body: { hearing: updatedHearing } }).then((res) => {
-      setHearing(res.hearing);
+      setHearing(res?.hearing);
```

Comment:
are you sure about setting potentially an `undefined` hearing ?

is it not better to retain old state of hearing if any request fails and res becomes null/undefined ?